src/Domain/Entity/Network.php

```diff
@@ -173,4 +178,14 @@ public function isRadio(): bool
     {
         return ($this->getMedium() === NetworkMediumEnum::RADIO);
     }
+
+    public function getOptions(): Options
```

Comment:
Options shall be a stored as a json column on the network table, it shall always be returned. It might be an empty, in which case an empty Options object shall be returned